index.html

```diff
@@ -0,0 +1,65 @@
+    <header id="header" class="wrapper-sm lg:wrapper-lg"></header>
+    <main id="main">
+      <section id="hero" class=" wrapper-sm lg:wrapper-lg "></section>
+      <section id="icon-section"></section>
+      <section id="managedTeams" class=""></section>
+      <section id="unsere-teams" class="wrapper-sm lg:wrapper-lg"></section>
+      <section id="skillset-section" class="space-y-14 md:space-y-28 wrapper-sm lg:wrapper-lg">
+        <div id="skillset" class=" text-white space-y-8">
+          <div id="textWithLine" class="relative h-[46px]">
+               <h1 class="absolute font-ralewayBold text-[20px] md:text-[26px] z-0 text-white">Skillset unserer Entwickler:innen</h1>
+               <div class="relative bg-primary w-[92.24px] h-[8px] -rotate-[3.11deg] top-[26px] left-5 z-10 opacity-[80%]"></div>
+               <div class="relative bg-primary w-[92.24px] h-[8px] -rotate-[3.11deg] top-[26px] left-5 z-10 opacity-[10%]"></div>
+           </div>
+           <div id="skills-btn" class="flex flex-wrap md:flex-wrap-reverse gap-x-2 gap-y-3"></div>
+           <h1 class="text-primary font-robotoBlack text-[16px] tracking-wider">ZUM TECH-SKIlLPROFIL</h1>
+       </div>
+       <div id="testimonials" class="p-4">
+          <div id="textWithLine" class="relative h-[46px] ">
+          <h1 class="absolute font-ralewayBold text-[26px] z-10 text-white">Eine mögliche Teamkonstellation</h1>
+          <div class="relative bg-primary w-[230px] h-[8px] -rotate-[3.11deg] top-16 lg:top-6 left-2 lg:left-52 z-0 opacity-[70%] "></div>
+       </div>
+      
+      
+      </div>
+      <div class="grid grid-cols-1 mt-10 md:mt-32 lg:flex justify-center self-center lg:flex-col relative lg:p-8">
+        <img src="./src/Assets/media/Ojembees2.webp" alt="Ojemba Team" class="w-[80%] self-center p-8 wrapper-lg ">
+        <section class="-mt-12 lg:w-[85%]  lg:mt-0 md:container md:mx-auto lg:absolute -bottom-8 lute lg:bottom-14 text-white lg:flex gap-y-6 gap-x-4 px-8 py-4 backdrop-opacity-8 backdrop-invert bg-secondary/90 md:gap-20 md:px-20 md:py-8 md:bottom-0 md:self-center lg:border-2 border-black">
+          <article class="card lg:w-[calc(100%/3)]">
+            <h5 class="font-montBold text-[16px] md:text-[20px] mb-[10px] md:mb-[20px]">Arnold:</h5>
+            <p class="font-montRegular italic text-[14px] md:text-[16px]">“d diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam“</p>
+          </article>
+          <article class="card lg:w-[calc(100%/3)]">
+            <h5 class="font-montBold text-[16px] md:text-[20px] mb-[10px] md:mb-[20px]">Clever:</h5>
+            <p class="font-montRegular italic text-[14px] md:text-[16px]">“d diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam“</p>
+          </article>
+          <article class="card lg:w-[calc(100%/3)]">
+            <h5 class="font-montBold text-[16px] md:text-[20px] mb-[10px] md:mb-[20px]">Izedin:</h5>
```

Comment:
Thanks Some of them they are not in tailwindcss like 48px so I decided to user rem instead

Reply:
hey, @Bateyjosue  48px If it's about font size you will have text-5xl If it's other spacing it will  be 12  you can use Tailwind documentation to check if it exists or not.